libguard/guard_file.cpp

```diff
@@ -12,33 +13,26 @@ namespace openpower
 namespace guard
 {
 using namespace openpower::guard::log;
+using namespace openpower::guard::exception;
 
 GuardFile::GuardFile(const fs::path& file) : guardFile(file)
 {
-    try
```

Comment:
Why you are removing the try and catch block for ifstream operation? I think it will throw an exception. Please make sure whether the using ifstream (or its base) class or function will throw an exception or not.

Reply:
Changed to original try catch block.